src/fides/api/service/connectors/sql_connector.py

```diff
@@ -529,7 +550,14 @@ def create_client(self) -> Engine:
     # Overrides SQLConnector.query_config
     def query_config(self, node: ExecutionNode) -> BigQueryQueryConfig:
         """Query wrapper corresponding to the input execution_node."""
-        return BigQueryQueryConfig(node)
+
+        db: Session = Session.object_session(self.configuration)
+        namespace_meta: Optional[BigQueryNamespaceMeta] = None
+
+        if raw_meta := SQLConnector.get_namespace_meta(db, node.address.dataset):
+            namespace_meta = BigQueryNamespaceMeta(**raw_meta)
```

Comment:
right so what i'm thinking is that instead of "casting" the raw_meta here, we'd cast it _within_ the scope of the `BigQueryQueryConfig`, such that we could initialize a generic `namespace_meta` attribute _generically_ on the `SQLConnector` base class

Reply:
I updated it to this
```
db: Session = Session.object_session(self.configuration)
        return BigQueryQueryConfig(
            node, SQLConnector.get_namespace_meta(db, node.address.dataset)
        )
```
The `SQLLikeQueryConfig` super class validates the meta dict based on the `namespace_meta_schema` defined at the `BigQueryQueryConfig`
```
class SQLLikeQueryConfig(QueryConfig[T], ABC):
    """
    Abstract query config for SQL-like languages (that may not be strictly SQL).
    """

    namespace_meta_schema: Optional[Type[NamespaceMeta]] = None

    def __init__(self, node: ExecutionNode, namespace_meta: Optional[Dict] = None):
        super().__init__(node)
        self.namespace_meta: Optional[NamespaceMeta] = None

        if namespace_meta is not None:
            if self.namespace_meta_schema is None:
                raise MissingNamespaceSchemaException(
                    f"{self.__class__.__name__} must define a namespace_meta_schema when namespace_meta is provided."
                )
            try:
                self.namespace_meta = self.namespace_meta_schema.model_validate(
                    namespace_meta
                )
            except ValidationError as exc:
                raise ValueError(f"Invalid namespace_meta: {exc}")
```